internal/configs/harmony/harmony.go

```diff
@@ -27,6 +27,7 @@ type HarmonyConfig struct {
 	Revert     *RevertConfig     `toml:",omitempty"`
 	Legacy     *LegacyConfig     `toml:",omitempty"`
 	Prometheus *PrometheusConfig `toml:",omitempty"`
+	Feature    *FeatureConfig    `toml:",omitempty"`
```

Comment:
I would suggest to add it to `GeneralConfig`. We shall make each config in `HarmonyConfig` for a micro-service within the node codebase. In this case, `Pruning` itself is not a service, but a part of `BlockChain`.

Reply:
You are right, edited.